modules/programs/ssh.nix

```diff
@@ -234,6 +243,14 @@ in
       '';
     };
 
+    certificateFile = mkOption {
```

Comment:
From what I can tell this option is not actually used anywhere. I think you could just remove this since I suspect it is not very common to set a certificate file in the wildcard section.

Reply:
Hmm, did you resolve this by mistake? I think this option still is not used, although I still think it is best to simply remove it 🙂

Reply:
> Hmm, did you resolve this by mistake?
Looks like. I've never really used GH for anything work related. I expected it to drop this hunk for me.